plugins/feeds/public/benkow_rat.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from datetime import datetime, timedelta
+import csv
+import logging
+
+from core.observables import Url, Ip
+from core.feed import Feed
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+from core.errors import GenericYetiError
+
+class BenkowTrackerRat(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=12),
+        "name": "BenkowTrackerRat",
+        "source" : "http://benkow.cc/export_rat.php",
+        "description": "This feed contains known Malware C2 servers",
+    }
+
+    def update(self):
+        resp = requests.get(self.source, proxies=yeti_config.proxy)
+        if not resp.ok:
+            raise GenericYetiError("{} - got response code {}".format(self.name, resp.status_code))
+        
+        reader = csv.reader(resp.content.strip().splitlines(), delimiter=';', quotechar='"')
+        for line in reader:
+            self.analyze(line)
+
+    def analyze(self, line):
+        if line[0] == 'id':
+            return
+
+        ID, Family, Url, IP, First_Seen , blank = line
```

Comment:
You can use `_` for any variables that you're not planning to use later on.

```suggestion
        _id, family, url, ip, first_seen, _ = line
```

Also please always use lowercase for variable names (whenever possible). In this case, `Url` was overriding the `Url` you're importing and calling in L48 :D